html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+        <br>
+        <input type="text" id="email" name="email">
+      </div>
+      <div>
+        <label for="reasons">Why is coding cool?</label>
+        <br>
+        <select name="reasons" id="reasons">
+          <option value="Select an otpion" selected="selected">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option avlue="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label>Favorite Language (choose one)</label>
```

Comment:
This doesn't need to be a `label` element, since it is not labelling a specific form control element.